src/__tests__/starWarsSchema.ts

```diff
@@ -135,6 +133,7 @@ const characterInterface = new GraphQLInterfaceType({
     }
 
     // Not reachable. All possible types have been considered.
+    // istanbul ignore next (unreachable)
```

Comment:
They should be added by our custom babel plugin can you make it so it is run after TS?

Reply:
I'll check. Thanks!

Reply:
@dotansimha On second thought, we have only few dozen places with `invariant` inside tests, and most of the time it's just to test that certain function is never called which is better tested by specialized mock libraries. It stupid to run babel just to insert these comments.
So it's perfectly fine to disable Istanbul here, just try to merge two comments together like so:
```js
 // istanbul ignore next (Not reachable. All possible types have been considered.)
```